config/init/common/lxc-net.in

```diff
@@ -26,8 +27,11 @@ LXC_IPV6_NAT="false"
 
 [ ! -f $distrosysconfdir/lxc ] || . $distrosysconfdir/lxc
 
-use_iptables_lock="-w"
-iptables -w -L -n > /dev/null 2>&1 || use_iptables_lock=""
+NFT="$(which nft)"
+if [ -z "$NFT" ] || ! nft list ruleset > /dev/null 2>&1 || [ "$LXC_USE_NFT" != "true" ]; then
```

Comment:
@pablofsf it feels like this line and the similar two lines that are `[ -n "$NFT" ] && nft list ruleset > /dev/null 2>&1 && [ "$LXC_USE_NFT" = "true" ];` could do with being moved into a function called something like `useNftables()` that way the logic governing the use of nftables is kept in one place and it would read a bit easier when used.